modules/mod-deep-learning/ExploreHuggingFaceDialog.cpp

```diff
@@ -0,0 +1,62 @@
+/**********************************************************************
+
+   Audacity: A Digital Audio Editor
+   Audacity(R) is copyright (c) 1999-2021 Audacity Team.
+
+   ExploreHuggingFaceDialog.cpp
+   Hugo Flores Garcia
+
+******************************************************************/
+
+#include "ExploreHuggingFaceDialog.h"
+
+#include "Shuttle.h"
+#include "ShuttleGui.h"
+
+#include <wx/scrolwin.h>
+#include <wx/range.h>
+#include <wx/gauge.h>
+#include <wx/button.h>
+#include <wx/sizer.h>
+#include <wx/textdlg.h>
+#include <wx/stattext.h>
+#include <wx/hyperlink.h>
+
+// ExploreDialog
```

Comment:
No such class (any more?).  Remove this comment

Reply:
resolved in  09844aa67

Reply:
resolved in 09844aa67